.travis.yml

```diff
@@ -1,3 +1,4 @@
+sudo: false
```

Comment:
I looks like the containerized service that Travis uses for sudo:false builds doesn't require super privileges. I'm not certain of their implementation details but the tests are now passing.